common/current_preset.h

```diff
@@ -316,6 +316,30 @@ class CurrentPreset {
     if (!Load(preset)) Set(preset);
     LOCK_SD(false);
   }
+
+void SetStyle(int blade, const char* style) {
```

Comment:
I think this one is dangerous and we should not have it unless we really need it for some reason.

Reply:
this one is used throughout Edit Mode for styles, it was something you provided when we first began.

Reply:
I think we should remove this one and use the one below instead.

Reply:
ok, removed the void SetStyle(), will need to implement and test on Edit Mode version to see what changes are needed in prop.